docs/portal/transports/view-transport.md

```diff
@@ -1,6 +1,6 @@
 # View a Transport
 
-To view a transport, search the transport you want to view in the <strong> Transport </strong> page if it was not in the page, and then click the "Transport Name".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
+To view a transport, click the "Transport Name" of the transport entry in the list. If the transport was not in the current page, you can search it in the search box.
```

Comment:
Please change "search it" with "search for it."

Reply:
I'd also suggest  replacing "was" with "is"